app/admin/comment.rb

```diff
@@ -0,0 +1,53 @@
+ActiveAdmin.register Comment do
+  belongs_to :debate
+  actions :all, :except => [:show, :edit]
+
+  index do
+    selectable_column
+    id_column
+    column :content
+    column :status
+    column :created_at
+    actions do |comment|
+      if comment.accepted?
+        item "reject", reject_admin_debate_comment_path(debate, comment), method: :put, class: 'member_link'
+      elsif comment.pending?
+        item "accept", accept_admin_debate_comment_path(debate, comment), method: :put, class: 'member_link'
+        item "reject", reject_admin_debate_comment_path(debate, comment), method: :put, class: 'member_link'
+      else
+        item "accept", accept_admin_debate_comment_path(debate, comment), method: :put, class: 'member_link'
+      end
```

Comment:
Maybe we can change conditions not to duplicate logic and add `item 'reject'` if comment is not rejected and `item 'accept'` if comment is not accepted. This should cover all cases

~~~ruby
item('reject') if !comment.rejected?
item('accept') if !comment.accepted?
~~~